cognite_toolkit/_cdf_tk/client/testing.py

```diff
@@ -59,6 +61,10 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.lookup.extraction_pipelines = MagicMock(spec_set=ExtractionPipelineLookUpAPI)
         self.migration = MagicMock(spec=MigrationAPI)
         self.migration.mapping = MagicMock(spec_set=MappingAPI)
+        self.raw = MagicMock(spec=ExtendedRawAPI)
+        self.raw.databases = MagicMock(spec_set=RawDatabasesAPI)
+        self.raw.rows = MagicMock(spec_set=RawRowsAPI)
+        self.raw.tables = MagicMock(spec_set=RawTablesAPI)
```

Comment:
Unrelated to the rest of the PR, but forgot to update this when adding the `ExtendedRawAPI`. Risk-reviewer - please forgive me for not making a separate PR for these 6 lines.